site/mkdocs.yml

```diff
@@ -94,7 +94,8 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
-  - pymdownx.tabbed
+  - pymdownx.tabbed:
+      alternate_style: true
```

Comment:
What does this do?

Reply:
This is an alternate implementation of the tabbed view that works better on narrow devices (like phones).